src/components/Video.cpp

```diff
@@ -167,6 +167,12 @@ bool Video::setGeometry(unsigned width, unsigned height, unsigned maxWidth, unsi
       return false;
     }
   }
+  else if (_vertexArray)
+  {
+    Gl::disableVertexAttribArray(_vertexArray);
```

Comment:
Calling `disableVertexAttribArray` on `_vertexArray` is incorrect, because the argument is supposed to be an index representing a _vertex attribute_ of the currently bound vertex array, not the id of a vertex array.

I assume that the fix would still work if this call were removed, or does it not? If it doesn't it's probably working because of id collisions between the attributes and the vertex array itself.

Reply:
Yes, this works without the disable. Though, with my original changes (before moving it here) it didn't seem to.